### PR TITLE
solarish couple of strings fn.

### PR DESCRIPTION
**Pull request by devnexen**

### CODE REVIEW

**Review by JohnTitor — APPROVED** [review]

File: src/unix/solarish/mod.rs [hunk 1]
```diff
@@ -2800,6 +2800,15 @@ extern "C" {
...
     ) -> ::ssize_t;
+
+    pub fn strcasecmp_l(s1: *const ::c_char, s2: *const ::c_char, loc: ::locale_t) -> ::c_int;
+    pub fn strncasecmp_l(
+        s1: *const ::c_char,
+        s2: *const ::c_char,
+        n: ::size_t,
+        loc: ::locale_t,
+    ) -> ::c_int;
+    pub fn strsep(string: *mut *mut ::_char, delim: *const ::c_char) -> *mut ::c_char;
```
Comment (JohnTitor) [inline src/unix/solarish/mod.rs:2811]: Oh, missed a typo:

```suggestion
    pub fn strsep(string: *mut *mut ::c_char, delim: *const ::c_char) -> *mut ::c_char;
```

**Comment by JohnTitor** [comment]
@devnexen I think this is good to go once the conflict is resolved.

**Comment by devnexen** [comment]
oddily seems the change no longer pass here

**Pull request closed without merging**